pset-iterators.js

```diff
@@ -65,7 +72,11 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 2:')
 
 // Add your code below this line
+const presidentsNamedJames = presidents.filter(president => {
+  return president.president.includes('James')
```

Comment:
@anniecdavis27 nice use of `.includes()` to match against your targeted criteria